skeleton/win.cpp

```diff
@@ -1,4 +1,5 @@
 #ifdef _WIN32
+#define SDL_MAIN_HANDLED
```

Comment:
You need to add a call to `SDL_SetMainReady` in `main`/`WinMain`.
See https://wiki.libsdl.org/SDL_SetMainReady

Reply:
I just checked the implementation of [`SDL_SetMainReady`](https://github.com/libsdl-org/SDL/blob/c59d4dcd38c382a1e9b69b053756f1139a861574/src/SDL.c#L144-L148).
It looks like `SDL2main` is needed (for some platforms).

Reply:
This only seems to be needed on platforms that define `SDL_MAIN_NEEDED`, while Windows defines `SDL_MAIN_AVAILABLE` instead. For Windows, what `SDL2main` provides is equivalent to what both the skeleton and MinGW already provide, so calling `SDL_SetMainReady` shouldn't be necessary on Windows.